app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt

```diff
@@ -414,6 +416,14 @@ class Message : RealmObject, Snoozable {
         snoozeEndDate = flags.snoozeEndDate.toRealmInstant()
     }
 
+    fun getFormattedPreview(context: Context): String {
+        return when {
+            isEncrypted -> "\n" + context.getString(R.string.encryptedMessageHeader)
+            isReaction -> "\n" + context.getString(R.string.previewReaction, from.first().name, emojiReaction)
+            else -> preview.ifBlank { null }?.let { "\n${it.trim()}" } ?: ""
+        }
```

Comment:
Maybe it could make sense to factorize the common leading `\n`

Reply:
That's right, but I proposed another refactored version that doesn't add \n if the preview is blank.

Reply:
There is now a hardcoded `\n` in every returns of `getFormattedPreview()`, this is contradictory with what you just said.